multiversx_sdk/multisig/resources.py

```diff
@@ -0,0 +1,231 @@
+from typing import Any, Optional, Union
+
+from multiversx_sdk.abi.abi import Abi
+from multiversx_sdk.abi.biguint_value import BigUIntValue
+from multiversx_sdk.abi.fields import Field
+from multiversx_sdk.abi.interface import ISingleValue
+from multiversx_sdk.abi.multi_value import MultiValue
+from multiversx_sdk.abi.small_int_values import U64Value
+from multiversx_sdk.abi.string_value import StringValue
+from multiversx_sdk.abi.struct_value import StructValue
+from multiversx_sdk.core.address import Address
+from multiversx_sdk.core.code_metadata import CodeMetadata
+from multiversx_sdk.core.constants import ARGS_SEPARATOR
+from multiversx_sdk.core.tokens import TokenTransfer
+from multiversx_sdk.core.transactions_factory_config import TransactionsFactoryConfig
+from multiversx_sdk.smart_contracts.smart_contract_transactions_factory import (
+    SmartContractTransactionsFactory,
+)
+from multiversx_sdk.transfers.transfer_transactions_factory import (
+    TransferTransactionsFactory,
+)
+
+
```

Comment:
In JS, it seems there are no named constructors, and the corresponding logic is moved to the factories themselves.

https://github.com/multiversx/mx-sdk-js-core/blob/TOOL-329-add-mustisig-factory/src/multisig/resources.ts

Either way is fine, but perhaps we should have a similar implementation. I know it cannot be the same, of course (especially since the ABI components differ a bit).

Reply:
moved most of the logic out of the resources file.